app/controllers/VatRegistrationController.scala

```diff
@@ -22,6 +22,7 @@ import auth.VatTaxRegime
 import cats.data.OptionT
 import cats.instances.future._
 import config.FrontendAuthConnector
+import connectors.KeystoreConnector
```

Comment:
here seems to be just an import added without any code change requiring the import